commands/create_keypair.go

```diff
@@ -98,8 +98,9 @@ func addKeypair(cmd *cobra.Command, args []string) {
 	}
 
 	if apiResponse.StatusCode == 200 {
-		cleanID := util.CleanKeypairID(keypairResponse.Id)
-		msg := fmt.Sprintf("New key pair created with ID %s and expiry of %v hours", cleanID, keypairResponse.TtlHours)
+		msg := fmt.Sprintf("New key pair created with ID %s and expiry of %v",
+			util.Truncate(util.CleanKeypairID(keypairResponse.Id), 10),
+			util.DurationPhrase(int(keypairResponse.TtlHours)))
```

Comment:
You could use https://golang.org/pkg/time/#Duration.String

Reply:
That's a bit less friendly to read in my opinion

Thanks!